src/NzbDrone.Core/Music/AlbumRepository.cs

```diff
@@ -301,5 +302,13 @@ public Album FindByArtistAndName(string artistName, string cleanTitle)
                         .Where<Artist>(artist => artist.CleanName == cleanArtistName)
                         .SingleOrDefault(album => album.CleanTitle == cleanTitle);
         }
+
+        public Album FindAlbumByRelease(string releaseId)
+        {
+            string query = "SELECT * FROM Albums " +
+                         "WHERE Releases LIKE '%" + releaseId + "%'";
+
+            return Query.QueryText(query).FirstOrDefault();
```

Comment:
This function will also match on partial release Ids. May be better off to do with all with one LINQ statement: `return Query.FirstOrDefault(e => e.Releases.Any(r => r.Id == releaseId));`